protocol/avdl/keybase1/teamsearch.avdl

```diff
@@ -0,0 +1,18 @@
+@namespace("keybase.1")
+
+protocol teamSearch {
+    import idl "common.avdl";
+
+    record TeamSearchItem {
+        TeamID id;
+        string name;
+        union { null, string } description;
+        int memberCount;
```

Comment:
maybe add a `Time lastActive;` which could be from chat activity

Reply:
We talked about this in our quick meeting with @mmaxim -- the consensus was that all of the sorting would be done on the server side, so the client shouldn't need to know this unless it would be visually shown to the client in the search results.  Do you think it would be?

Reply:
we should a last active in the channel dialog. might be useful here as well and harmless to plumb down now if it makes the design better

Reply:
![image](https://user-images.githubusercontent.com/1144020/73024423-91c65000-3dfb-11ea-94d0-0692a5503b33.png)

Reply:
Cool will do, thanks.

Reply:
I guess this would be the max time of activity across all channels in the team.

Reply:
yeah which is what we'd use to sort anyway i think

Reply:
Cool.  Fine for now I'm sure, but over time people could try to game the results with bots posting in some hidden channel every second.  At some later date we'll need to figure something out more sophisticated.